src/Table/TableCell.spec.js

```diff
@@ -80,9 +80,8 @@ describe('<TableCell />', () => {
   });
 
   it('should render specified scope attribute even when in the context of a table head', () => {
-    const wrapper = shallow(<TableCell />);
+    const wrapper = shallow(<TableCell scope="row" />);
     wrapper.setContext({ table: { head: true } });
-    wrapper.setProps({ scope: 'row' });
```

Comment:
That is a weird refactoring :) I was actually trying to follow code styles as you have in this file as well (like `wrapper.setProps({ variant: 'head' });` below).